This patch is part of a series of patches for festival for
Debian GNU/Linux, to provide Debian specific site initialization

Index: b/lib/synthesis.scm
===================================================================
--- a/lib/synthesis.scm
+++ b/lib/synthesis.scm
@@ -68,7 +68,7 @@
    The default list of functions to be run on all synthesized utterances
    after Wave_Synth.  This will normally be nil but if for some reason you
    need to change the gain or rescale *all* waveforms you could set the
-   function here, in your siteinit.scm.")
+   function here, in your /etc/festival.scm.")
 
 (defvar after_synth_hooks default_after_synth_hooks
   "after_synth_hooks
